src/platform/EFR32/OTAImageProcessorImpl.h

```diff
@@ -67,6 +67,12 @@ class OTAImageProcessorImpl : public OTAImageProcessorInterface
     OTADownloader * mDownloader;
     OTAImageHeaderParser mHeaderParser;
     const char * mImageFile = nullptr;
+
+#define ALIGNMENT_BYTES 64
+    // Intermediate, word-aligned buffer for writing to the bootloader storage
+    static uint8_t writeBuffer[ALIGNMENT_BYTES];
+    // Offset indicates how far the write buffer has been filled
+    static uint16_t writeBufOffset;
```

Comment:
what guarantees that 2 separate OTAImageProcessor objects do not reuse the same buffer at the same time? Should there be a lock or should these be non-static? they only take under 100 bytes so non-static seems feasable.

Reply:
It is guaranteed by the fact that the EFR32 platform only creates a single instance of OTAImageProcessorImpl class and there is no plan to ever change that. If that does change at some point in the future then this will be revisited.

Reply:
Please add this in a comment. If you only ever create a single image of the processorimpl, there should be no harm/difference to not make these static right? That way the code would be obviously correct all the time.

Reply:
Added the comment. Members are static because the handler functions such as HandleProcessBlock() are static.